src/main/java/com/product/model/Customer.java

```diff
@@ -2,21 +2,27 @@
 
 import java.util.Date;
 
-import javax.validation.constraints.NotBlank;
+import lombok.Data;
 
+@Data
 public class Customer {
 
-	private int id;
+	public int custId;
```

Comment:
the data thats comming from customerService is called `custId` ????

if yes, then maybe in later PR we should change it to just `id`

Reply:
Done.